### PR TITLE
Error handling around multiple execution

### PR DESCRIPTION
**Pull request by ericdevg**
#182 @AndreSayreMSFT

This change fixed issues around multiple execution in such scenarios:
1. Start debugging while REPL is executing 
2. Try to execute one script from editor or sln windows while VS is already debugging another

### CODE REVIEW

File: PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs [hunk 1]
```diff
@@ -159,58 +160,58 @@ public void ClearBreakpoints()
...
+                if (_runspace.ConnectionInfo != null && Regex.IsMatch(commandLine, DebugEngineConstants.ExecutionCommandPattern))
                 {
-                    _callback = OperationContext.Current.GetCallbackChannel<IDebugEngineCallback>();
-                }
-            }
-            catch (Exception)
-            {
-                ServiceCommon.Log("No instance context retrieved.");
-            }
+                    Regex rgx = new Regex(DebugEngineConstants.ExecutionCommandFileReplacePattern);
```
Comment (ASayre) [inline PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs]: same comments re: Regex s :)

File: PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs [hunk 2]
```diff
@@ -159,58 +160,58 @@ public void ClearBreakpoints()
...
-            if (_runspace.RunspaceAvailability != RunspaceAvailability.Available)
-            {
-                _callback.OutputString(Resources.Error_PipelineBusy + Environment.NewLine);
+                    if (_mapLocalToRemote.ContainsKey(localFile))
+                    {
+                        commandLine = rgx.Replace(commandLine, _mapLocalToRemote[localFile]);
+                    }
+                    else
+                    {
+                        _callback.OutputString(string.Format(Resources.Error_LocalScriptInRemoteSession + Environment.NewLine, localFile));
```
Comment (ASayre) [inline PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs]: tangent: Perhaps a _callback.OutputLine would be useful

Comment (ericdevg) [inline PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs]: Fixed

File: PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs (hunk 1 above)
Comment (ericdevg) [inline PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs]: Made it as private field

**Comment by ASayre** [comment]
looks good :)

**Pull request merged**